applications/FluidDynamicsApplication/tests/TwoFluidMassConservationProcTest/3Dtest.json

```diff
@@ -17,33 +17,34 @@
         "material_import_settings"       	: {
```

Comment:
Same comment.

Reply:
Only a new line needs to be added in order to deactivate mass source term. I removed other changes.